charts/redpanda/Chart.yaml

```
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 5.0.9
+version: 5.0.10
```

This new feature seems worthy of a minor release, imo.
```suggestion
version: 5.1.0
```